lib/PuppeteerSharp/Page.cs

```diff
@@ -16,7 +16,7 @@ namespace PuppeteerSharp
     [DebuggerDisplay("Page {Url}")]
     public class Page : IDisposable
     {
-        public int DefaultNavigationTimeout { get; set; } = 30000;
+        public int DefaultNavigationTimeout { get; set; } = 30000000;
```

Comment:
30,000 seconds?

Reply:
oops :joy:

That was messing with my debugging. Good catch.